Products/CMFPlomino/PlominoForm.py

```diff
@@ -1055,9 +1055,25 @@ def openBlankForm(self, request=None):
                     self,
                     self.REQUEST).__of__(db)
         if (not invalid) or self.hasDesignPermission(self):
+            editmode=True
+            form_mode = request.get('plomino_form_mode')
+            if form_mode and form_mode == 'READ':
+                editmode=False
+            if form_mode and form_mode == 'WRITE':
```

Comment:
For consistency sake, I would prefer `EDIT` instead of `WRITE` (we have that convention for field templates for instance)